pool/message.go

```diff
@@ -597,8 +597,14 @@ func GenerateSolvedBlockHeader(headerE string, extraNonce1E string,
 	extraNonce2E string, nTimeE string, nonceE string, miner string) (*wire.BlockHeader, error) {
 	headerEB := []byte(headerE)
 
+	log.Debugf("header is %s\n", headerE)
```

Comment:
Are these `\n` new lines temporary?

Reply:
Yeah they are, I suspected the solved block reconstruction might not be the same as the cpu miner's, outputting them here solely for the values.